taskRequests/details/script.js

```diff
@@ -157,11 +161,21 @@ async function approveTaskRequest(userId) {
     });
 
     if (res.ok) {
+      showToast('Task Approved Successfully', 'success');
       taskRequest = await fetchTaskRequest();
       requestorsContainer.innerHTML = '';
       renderRequestors(taskRequest?.requestors);
+    } else {
+      showToast(
+        'The requested operation could not be completed. Please try again later.',
+        'failure',
```

Comment:
Can we move this text message in constants?

Reply:
Is it only used here,  should we consider adding it to our constants.

Reply:
You are using two times here.

Reply:
Done